frontend-react/package.json

```diff
@@ -177,11 +179,10 @@
         "@types/testing-library__jest-dom": "^5.14.2",
         "@typescript-eslint/eslint-plugin": "^5.9.1",
         "@typescript-eslint/parser": "^5.9.1",
-        "eslint": "^7.26.0",
+        "eslint": "^8.12.0",
```

Comment:
Major version bump - any breakages we should be concerned about? Everything seems okay locally and there doesn't seem to be anything tripping up in the build pipeline. Just trying to think through silent failure scenarios.

Reply:
There were breakages. The configuration changes and the one code edit to remove React were part of the fixes. 

It's just the linter, so I checked:
 - that it catches mistakes
 - that it doesn't generate false positives by running against our whole project

Reply:
I added a `to test` section to the issue. I should have included it, originally, sorry.